apps/builder/app/builder/features/settings-panel/settings-panel.tsx

```diff
@@ -20,9 +20,7 @@ export const SettingsPanel = ({ selectedInstance }: SettingsPanelProps) => {
           defaultValue={selectedInstance.label}
           onChange={(event) => {
             const { value } = event.target;
-            if (value.trim()) {
-              set({ setting: "label", value });
-            }
+            value.trim() ? set("label", value) : set("label", undefined);
```

Comment:
Why not if/else? Ternary is harder to understand when operands effectful

Reply:
I hate else's haha, but I can make the ternary effectless...